buzzmobile/simulation/sim_car_interface/src/sim_car_interface.cpp

```diff
@@ -35,13 +35,13 @@ double speed_measured_right = 0.0;
 double steer_set_point = 0.0;
 
 // TODO(sahit): change to rosparams
-constexpr double chassis_length = 0.33246;
-constexpr double chassis_width = 0.28732;
+constexpr double chassis_length = 1.8; 
+constexpr double chassis_width = 1.57;
```

Comment:
Wait hold on this doesn't at all match https://github.com/gtagency/buzzmobile/blob/master/buzzmobile/constants.yaml#L39 (which also says our wheels are 2.2 meters in circumference, which is wrong)